project-base/tests/App/Functional/Model/Product/Availability/ProductAvailabilityCalculationTest.php

```diff
@@ -38,36 +38,43 @@ class ProductAvailabilityCalculationTest extends FunctionalTestCase
      * @param mixed $usingStock
      * @param mixed $stockQuantity
      * @param mixed $outOfStockAction
-     * @param \Shopsys\FrameworkBundle\Model\Product\Availability\Availability|null $availability
-     * @param \Shopsys\FrameworkBundle\Model\Product\Availability\Availability|null $outOfStockAvailability
-     * @param \Shopsys\FrameworkBundle\Model\Product\Availability\Availability|null $defaultInStockAvailability
-     * @param \Shopsys\FrameworkBundle\Model\Product\Availability\Availability|null $expectedCalculatedAvailability
+     * @param string|null $availability
```

Comment:
I do not see need for this whole commit

Reply:
The reason is that there are no injected services in the data provider.